app/controllers/api/user_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UseController < ApplicationController
```

Comment:
should be `class Api::UsersController < ApiController`, dont forget to inherit your controller in `ApiController` since you deliberately created one for all of your API calls